### PR TITLE
Fix the bug that websocket-url was not propagated to components

### PR DESCRIPTION
**Pull request by d-hayashi**
## What?
Fix the bug that websocket-url was not propagated from webviz to the newly implemented components

## Why?
Bug fix

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/webviz/pull/8#" title="Author self-approved">d-hayashi</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fwebviz).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/dataware-tools/webviz/blob/dataware-tools/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by WatanabeToshimitsu — APPROVED** [review]
LGTM!

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/webviz/pull/8#" title="Author self-approved">d-hayashi</a>*, *<a href="https://github.com/dataware-tools/webviz/pull/8#pullrequestreview-869977190" title="Approved">WatanabeToshimitsu</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fwebviz).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/dataware-tools/webviz/blob/dataware-tools/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
